ledger/sandbox-on-x/src/classic/scala/com/daml/platform/ledger/SandboxServer.scala

```diff
@@ -99,29 +101,34 @@ final class SandboxServer(
     }
   }
 
+  private def uploadPackages(
+      writeService: WriteService
+  )(file: File)(implicit executionContext: ExecutionContext): Future[Unit] = {
+    val submissionId = Ref.SubmissionId.assertFromString(UUID.randomUUID().toString)
+    for {
+      dar <- Future.fromTry(DarParser.readArchiveFromFile(file).toTry)
+      _ <- writeService
+        .uploadPackages(submissionId, dar.all, None)(
+          LoggingContext.ForTesting,
+          NoOpTelemetryContext,
```

Comment:
This operation is a fire-and-forget. It doesn't wait for the package to actually appear on the Ledger API so the packages can still be uploaded concurrently in the indexer. One way to do it would be to upload and then wait until they appear on the `IndexService.packageEntries` stream.